WinUIGallery/ControlPagesSampleCode/ContentDialog/ContentDialogSample1_cs.txt

```diff
@@ -1,16 +1,18 @@
 ﻿private async void ShowDialog_Click(object sender, RoutedEventArgs e)
 {
-    ContentDialog dialog = new ContentDialog();
-
-    // XamlRoot must be set in the case of a ContentDialog running in a Desktop app
-    dialog.XamlRoot = this.XamlRoot;
-    dialog.Style = Application.Current.Resources["DefaultContentDialogStyle"] as Style;
-    dialog.Title = "Save your work?";
-    dialog.PrimaryButtonText = "Save";
-    dialog.SecondaryButtonText = "Don't Save";
-    dialog.CloseButtonText = "Cancel";
-    dialog.DefaultButton = ContentDialogButton.Primary;
-    dialog.Content = new ContentDialogContent();
-
+    ContentDialog dialog = new()
+    {
+        Title = "Save your work?",
```

Comment:
The setting of Style was lost.  This will cause the dialog to have square, not rounded corners.